src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	//implement Person and Tag classes
+	Person people;
+	Tag tag;
+	String ADDED = "abc";
+	String DELETED = "def";
+	public Name getname(){
```

Comment:
The point of an association class is to describe an association between two classes. It is a stand-alone class itself with pointers to the classes, whose association it is describing. 

You must pass in Tag , Person instances and an Operation( add/remove) to the Tagging constructor. 

Returning a class variable with a constant value assigned to it is what you are doing here. This is not the expected implementation for an Association class.